README.md

```diff
@@ -1 +1,76 @@
-# Leaderboard-Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard Project
+
+> In this project I have set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. I have developed a first working version of the app following a wireframe, but without styling -Focus is on functionality. I have consumed the Leaderboard API using JavaScript async and await and added some styling.
+
+
+###### Languages 
+- HTML
+- CSS
+- JAVASCRIPT
+###### Tools  
+- Github
+###### Code Editor
+- VS code
+###### Packages 
+- NPM
+- Node
+- Webpack
+
+## Live Demo (if available)
+
+[View Live Demo Here](https://rn486.github.io/Leaderboard-Project/)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@Rn486](https://github.com/Rn486)
+- Twitter: [@recillahk](https://twitter.com/recillahk)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/recillah-khamala-071151b7/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Rn486/Leaderboard-Project/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+- 
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [OPTIONAL] Please update your Readme, it will be nice to update this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional. Check out this [template](https://github.com/microverseinc/readme-template) for more clarifications